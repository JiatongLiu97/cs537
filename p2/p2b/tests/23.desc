Alias <alias-name> to echo replacement for alias-name. 
