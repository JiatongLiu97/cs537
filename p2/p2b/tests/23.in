alias ls /bin/ls
alias ll /bin/ls -l
alias ls
alias ll
exit
