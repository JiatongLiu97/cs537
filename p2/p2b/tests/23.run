./mysh tests/23.in
