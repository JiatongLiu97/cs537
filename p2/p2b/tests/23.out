alias ls /bin/ls
alias ll /bin/ls -l
alias ls
ls /bin/ls
alias ll
ll /bin/ls -l
exit
